13_biblioteki/13_1_BLAS.tex

```diff
@@ -0,0 +1,68 @@
+% !TEX encoding = UTF-8 Unicode
+\section{BLAS - Basic Linear Algebra Subprograms}
+%%%%%%%%%%%%%%%%
+	\begin{frame}{Levels}
```

Comment:
Maybe it would be a good idea to name frames which start particular sections the same as those sections? For example here, instead of 'Levels', 'BLAS - Basic Linear Algebra Subprograms'. I know it's redundant, but I think it will be more readable.